source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -76,6 +76,17 @@ You could also connect a customer-relationship management (CRM) or case manageme
 
 Read more about [reporting](/reporting/#reporting) and [refunding payments](/refunding_payments/#refunding-payments).
 
+### When you'll receive payments
+
+You'll receive a payment in your bank account within 2 business days of your user completing their payment.
```

Comment:
This is probably true but some orgs have different arrangements with their PSP. This might be 2 business days, or might be more. e.g. FCO get weekly payouts

Reply:
I did consider adding this but I wonder if it's necessary? Because you'll know if you've set up a different arrangement with your PSP.

Reply:
Think we're ok on this one following our chat, so resolving the conversation.